### PR TITLE
feat(create-medusa-app): add database options

### PR DESCRIPTION
**Pull request by shahednasser**
## What

Adds new options for easier usage of the `create-medusa-app` command for regular medusa users.

The following options are added:

- `--skip-db`: Skips creating the database, running migrations, and seeding, and subsequently skips opening the browser. Useful if the developer wants to set the database URL at a later point in the configurations.
- `--db-url <url>`:  Skips database creation and sets the database URL to the provided URL. Throws an error if can't connect to the database. Will still run migrations and open the admin after project creation. Useful if the developer already has database created, locally or remotely.
- `--no-migrations`: Skips running migrations, creating admin user, and seeding. If used, it's expected that you pass the `--db-url` option with a url of a database that has all necessary migrations. Otherwise, unexpected errors will occur. Helpful only if combined with `--db-url`
- `--no-browser`: Disables opening the browser at the end of the project creation and only shows success message.
- `--directory-path <path>`: Allows specifying the directory path to install the project in. Useful for testing.

### CODE REVIEW

File: packages/create-medusa-app/src/index.ts [hunk 1]
```diff
@@ -14,6 +14,29 @@ program
     "--stable",
     "Install the latest stable version. This removes all onboarding features"
   )
+  .option(
+    "--skip-db",
+    "Skips creating the database, running migrations, and seeding, and subsequently skips opening the browser.",
```
Comment (olivermrbl) [inline packages/create-medusa-app/src/index.ts:19]: question(non-blocking): Why are we skipping browser opening here?

File: packages/utils/package.json [hunk 2]
```diff
@@ -29,7 +29,6 @@
   },
   "dependencies": {
     "awilix": "^8.0.1",
-    "glob": "^7.1.6",
```
Comment (olivermrbl) [inline packages/utils/package.json:32]: nice!

**Review by olivermrbl — APPROVED** [review]
LGTM!

File: packages/create-medusa-app/src/index.ts (hunk 1 above)
Comment (shahednasser) [reply packages/create-medusa-app/src/index.ts:19]: Because the database wasn't created, so the backend isn't really usable.

Comment (olivermrbl) [reply packages/create-medusa-app/src/index.ts:19]: My bad, I thought of a case where you pass --db-url, but I guess it is safe to assume that you would use one or the other. Discard the comment.

**Pull request merged**